rackcity/models/asset.py

```diff
@@ -77,3 +78,28 @@ def save(self, *args, **kwargs):
                         self.asset_number = asset_number
                         break
             super(Asset, self).save(*args, **kwargs)
+            self.add_network_ports()
+            self.add_power_ports()
```

Comment:
same as for racks....I think we need similar code on asset.delete() so that the asset's associated rows in the networks ports & power ports tables are also deleted?

Reply:
See above